drivers/storage/portworx/util/util.go

```diff
@@ -36,6 +37,8 @@ const (
 	DefaultOpenshiftStartPort = 17001
 	// PortworxSpecsDir is the directory where all the Portworx specs are stored
 	PortworxSpecsDir = "/configs"
+	// PortworxTLSCertsDir is where all tls certs are to be placed
+	PortworxTLSCertsDir = "/etc/pwx/"
```

Comment:
Once we go this route it won't be easy to change and remove the default base path for existing customers. We won't know whether we need to add the base path or not.
I feel keeping it open-ended is safer in case the users are unwilling to move/copy their certificates in /etc/pwx. Also, it could be some automation in their infra that puts certificates in a location which is not /etc/pwx. The only downside  is that users will have to provide the full path every time.

Reply:
> Once we go this route it won't be easy to change and remove the default base path for existing customers. We won't know whether we need to add the base path or not.
> I feel keeping it open-ended is safer in case the users are unwilling to move/copy their certificates in /etc/pwx. Also, it could be some automation in their infra that puts certificates in a location which is not /etc/pwx. The only downside is that users will have to provide the full path every time.

As discussed on slack: we will merge it into a feature branch separate from master until we've made a decision to ship TLS changes, so customer backwards compat issues will not arise. Will address https://portworx.atlassian.net/browse/OPERATOR-299  in a separate PR